apps/src/maze/CollectorGemCounter.jsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import color from '../util/color';
+import msg from './locale';
+
+const styles = {
+  container: {
+    display: 'inline-block',
+    fontFamily: '"Gotham 5r"',
+    paddingRight: 10,
+    pointerEvents: "none",
+    textAlign: 'right',
+    verticalAlign: 'top',
+  },
+  label: {
+    color: color.lighter_gray,
+    textTransform: 'capitalize'
+  },
+  hr: {
+    borderColor: color.lighter_gray,
+    borderWidth: 2,
+    margin: '0 auto',
+  },
+  gemImage: {
+    backgroundSize: "100%",
+    display: "inline",
+    padding: 5
+  },
+  checkmark: {
+    color: 'lightgreen',
+    fontSize: 18,
+    left: 5,
+    lineHeight: '25px',
+    position: 'relative',
+    top: 7,
+  },
+};
+
+export class CollectorGemCounter extends React.Component {
```

Comment:
yeah, but then I'd be polluting `common.js` with a collector-specific phrase _just_ to add a storybook entry.

I did a bit of looking around, and it turns out _none_ of our React components with app-specific translations have Storybook entries, probably for this same reason. I think the solution to that is probably to add a tech debt item to look into why storybook is being a grouch about this.